projects/storefrontapp-e2e-cypress/cypress/integration/mobile/my-account/close-account-mobile.e2e-spec.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  closeAccountTest,
+  verifyAsAnonymous,
+} from '../../../helpers/close-account';
+import { formats } from '../../../sample-data/viewports';
+import * as homepage from '../../../helpers/homepage';
+
+describe(`${formats.mobile.width + 1}p resolution - Close Account page`, () => {
+  before(() => {
+    cy.window().then(win => win.sessionStorage.clear());
+    cy.viewport(formats.mobile.width, formats.mobile.height);
+  });
+
+  verifyAsAnonymous();
+
+  beforeEach(() => {
+    cy.viewport(formats.mobile.width, formats.mobile.height);
+  });
+
+  describe('close account test for logged in user', () => {
+    before(() => {
+      cy.viewport(formats.mobile.width, formats.mobile.height);
+      homepage.clickHamburger();
+    });
+
+    beforeEach(() => {
+      cy.restoreLocalStorage();
+      cy.viewport(formats.mobile.width, formats.mobile.height);
```

Comment:
we don't need this line as we have it in line 17, which will do it and work still